### ISSUE TITLE
[Question] dependencies confliction in version

### ISSUE THREAD

**Issue by r-y1** [PKU-MARL/omnisafe#196]
### Required prerequisites

- [X] I have read the documentation <https://omnisafe.readthedocs.io>.
- [X] I have searched the [Issue Tracker](https://github.com/PKU-MARL/omnisafe/issues) and [Discussions](https://github.com/PKU-MARL/omnisafe/discussions) that this hasn't already been reported. (+1 or comment there if it has.)
- [X] Consider asking first in a [Discussion](https://github.com/PKU-MARL/omnisafe/discussions/new).

### Questions

While I install omnisafe from PyPI using `pip install omnisafe`, there are some dependency conflicts.
Does it matter?
```bash
ERROR: pip's dependency resolver does not currently take into
account
al1 the packaces
that are
installed. This behaviour is the source of the following dep
pandas-profiling 3.2.0 requires joblib-=1.1.0, but you have joblib 1.2.0 which is incompatible.
```

**Comment by rockmagma02** [comment]
Thank you for your feedback, but please provide the package list used in your conda virtual environment and your python version, and other information so that we can reproduce your problem.

As the quickest solution, we strongly recommend that you install omnisafe into a new clean virtual environment to prevent conflicts between existing packages in your virtual environment and omnisafe dependencies.

**Comment by r-y1** [comment]
Well, I installed omnisafe in the tutorial of omnisafe, you can checkout it.

**Comment by muchvo** [comment]
I have reproduced it. We will fix it in the later version, thanks for your feedback.

**Comment by muchvo** [comment]
fixed in #197
